app/controllers/suggested_edit_controller.rb

```diff
@@ -0,0 +1,98 @@
+class SuggestedEditController < ApplicationController
+  before_action :set_suggested_edit, only: [:show, :approve, :reject]
+
+  def show
+    render layout: 'without_sidebar'
+  end
+
+  def approve
+    return unless @edit.active?
```

Comment:
`return` is a bad idea here: it'll leave the request unterminated. Use `not_found` instead to render a 404:

```suggestion
    not_found unless @edit.active?
```

Again, likewise for anywhere else you've done a `return` without rendering or 404ing.